draft-ietf-scitt-architecture.md

```diff
@@ -228,9 +228,11 @@ Over time, an Issuer may register new Signed Statements about an Artifact in a T
 Receipt:
 
 : a cryptographic proof that a Signed Statement is included in the Append-only Log.
-Receipts are based on Signed Inclusion Proofs, such as those as described in COSE Signed Merkle Tree Proofs {{-COMETRE}};
-they can be built on different verifiable data structures, not just binary merkle trees.
-A Receipt consists of a Transparency Service-specific inclusion proof for the Signed Statement, a signature by the Transparency Service of the state of the Append-only Log after the inclusion, and additional metadata (contained in the signature's protected headers) to assist in auditing.
+Receipts demonstrate the inclusion of Signed Statements in the Append-only Log of a Transparency Service and have a specific abstract role in Transparency as defined in this document.
+A Receipt combines a signature by the Transparency Service of the state of the Append-only Log after the inclusion with additional metadata (contained in the signature's protected headers) to assist in auditing.
```

Comment:
A Receipt consists of a Transparency Service-specific inclusion proof for the Signed Statement, a signature by the Transparency Service of the state of the Append-only Log after the inclusion, and additional metadata (contained in the signature's protected headers) to assist in auditing.

Reply:
I think, we should stick with the Original text here, which was much crisp and clear!

Can i please ask, why the need for this re-alignment? What is the ask ?

Reply:
I mean Line 232 can be replaced with the original text from left!

Rest all re-alignment can be taken from your PR

Reply:
Yep, the ask was this issue: https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/276 it was specifically asking if a Receipt was just an inclusion proof or something else.

On that issue we discussed fore shadowing the definition of transprency, which defines receipts as, essentially, signed inclusion proofs.

Reply:
I understand the concerne was that the lack of clarity on that made it unclear whether or how other proof types, specific to the implementation, should be described. In this pr I'm just saying 'not receipts'

Reply:
I think reducing the entire defintion to just

"Receipts demonstrate the inclusion of Signed Statements in the Append-only Log of a Transparency Service and have a specific abstract role in Transparency as defined in this document."

And nothing else, would address the original issue. Receipts can't both be an abstract notion and also a specific prescription imo

Reply:
@yogeshbdeshpande a "less is more" update based on your review I think